boards/arm/nrf51_ble400/Kconfig.board

```diff
@@ -0,0 +1,7 @@
+# Kconfig - nRF51 BLE400 board configuration
```

Comment:
do you want to have a copyright line?

Reply:
not sure what copyright to put there

Reply:
added copyrights

Reply:
Example:  
 > Copyright (c) 2018 Zephyrus Zephyr Inc.

Refer to http://docs.zephyrproject.org/contribute/contribute_guidelines.html

>A license tells you what rights you have as a developer, as provided by the copyright holder. It is important that the contributor fully understands the licensing rights and agrees to them. Sometimes the copyright holder isn’t the contributor, such as when the contributor is doing work on behalf of a company.

Reply:
I still dont see copyright line in this file.

Reply:
Sorry - updated now